test/support/server.js

```diff
@@ -17,7 +42,15 @@ app.all('/url', (req, res) => {
 });
 
 app.all('/echo', (req, res) => {
-  res.writeHead(200, req.headers);
+  let headers = req.headers;
+  if(process.env.EXPOSE_HTTP2){
+    Object.keys(headers).forEach(function(name) {
+      if(isPseudoHeader(name)){
+        delete headers[name];
```

Comment:
Maybe this should translate the header to HTTP/1 header, or allow both in the test using it?

Reply:
HTTP/2 use some headers to control protocol(e.g. :method, :authority, :path ...) and it can not set by user. I just delete those headers here.